### DISCUSSION TITLE
2022年2月以前から更新していない古い記事の更新ができない

### DISCUSSION THREAD

**Discussion by katzueno** [increments/qiita-discussions#608] [要望・アイデア・不具合]
お世話になります。

（おそらく）2022年2月24日以前から更新していない記事の更新ができないです。

- 記事を編集する時に「記事のプレビューをご確認ください」![image](https://github.com/increments/qiita-discussions/assets/485751/6f4c598f-24ea-443e-b317-42e897d8c3da) のポップアップが表示される記事を編集しようとする
- 「エラーにより投稿できませんでした。何度も続く場合はサポートにお問い合わせください。」![image](https://github.com/increments/qiita-discussions/assets/485751/c3d864fa-11a3-40d9-a47c-d3e35255d47b) とエラーが出て、記事の更新を公開できない

下書きの保存はできます。

**Comment by tanaka-asami415** [comment]
@katzueno 
ご報告ありがとうございます。
ご不便をおかけして申し訳ありません。調査して修正いたします🙇

**Reply by ohakutsu** [reply]
@katzueno 
お待たせしており申し訳ございません。
ご報告いただいたエラーは引き続き発生していますでしょうか。

こちら調査をしてみたのですが、上記のエラーを再現できず原因の特定ができておりません :bow: 
もし引き続き発生しておりましたら、[サポートフォーム](https://support.qiita.com/hc/ja/requests/new)からHARファイルを送っていただけないでしょうか。

HARファイルの取得方法は以下になります。
https://support.google.com/admanager/answer/10358597?hl=ja

**Comment by katzueno** [comment]
@ohakutsu お世話になります。
エラーは引き続き発生していました。
ブラウザも Safari & Arc 両方で発生しました。

さきほど、サポートフォーラムから両方のブラウザから取得した HAR ファイルをお送りいたしました。

よろしくお願いします。

**Reply by tanaka-asami415** [reply]
@katzueno 
早速ご対応いただきありがとうございます🙏
受領いたしました。引き続き調査いたします！

**Comment by ohakutsu** [comment]
@katzueno 
お待たせしました！
調査したところ、たまたま対象の記事の下書きでデータの不整合が起こっていることで、記事の更新が出来ないようでした。

データの不整合の解消には下書きの削除が必要なため、申し訳ないのですが、
一度、下書きの内容をバックアップを取ってから下書きを削除していただき、再度記事の更新を試していただけないでしょうか？
お手数をおかけしますが、よろしくお願いします！

**Comment by katzueno** [comment]
@ohakutsu ありがとうございます。

> 調査したところ、たまたま対象の記事の下書きでデータの不整合が起こっていることで、記事の更新が出来ないようでした。

なんと。たまたまだったのですね。

> 一度、下書きの内容をバックアップを取ってから下書きを削除していただき、再度記事の更新を試していただけないでしょうか？

実施したところ、解消しました。

ありがとうございました！
